plugins/Insights/lang/en.json

```diff
@@ -1,35 +1,35 @@
         "DayComparedToPreviousDay": "previous day",
-        "DayComparedToPreviousWeek": "same day in the previous week",
-        "DayComparedToPreviousYear": "same day in the previous year",
+        "DayComparedToPreviousWeek": "same day the week before",
+        "DayComparedToPreviousYear": "same day the year before",
         "Filter": "Filter",
-        "FilterIncreaserAndDecreaser": "Increaser & decreaser",
-        "FilterOnlyDecreaser": "Only decreaser",
+        "FilterIncreaserAndDecreaser": "Increasing and decreasing",
+        "FilterOnlyDecreaser": "Only decreasing",
         "FilterOnlyDisappeared": "Only disappeared",
-        "FilterOnlyIncreaser": "Only increaser",
+        "FilterOnlyIncreaser": "Only increasing",
         "FilterOnlyMovers": "Only movers",
         "FilterOnlyNew": "Only new",
         "IgnoredChanges": "Changes affecting less than %s visits were ignored.",
         "MonthComparedToPreviousMonth": "previous month",
-        "MonthComparedToPreviousYear": "same month in the previous year",
-        "MoversAndShakersWidgetTitle": "Movers and Shakers",
-        "NoResultMatchesCriteria": "No rows match the criteria",
+        "MonthComparedToPreviousYear": "same month the year before",
+        "MoversAndShakersWidgetTitle": "Movers and shakers",
+        "NoResultMatchesCriteria": "There are no such rows",
         "OverviewWidgetTitle": "Insights Overview",
-        "TitleConsideredInsightsChanges": "The rows increased or decreased by at least %1$s visits (%2$s%% of %3$s total visits).",
-        "TitleConsideredInsightsGrowth": "The following rows have a growth of at least %1$s%% compared to %2$s.",
-        "TitleConsideredMoversAndShakersChanges": "Considered movers only if they grew by more than %1$s%% visits or shrank by less than %2$s%% visits, new entries only if they increased by more than %3$s%% visits (%4$s), and disappeared rows if they shrank by less than %5$s%% visits (%6$s).",
-        "TitleConsideredMoversAndShakersGrowth": "%1$s changed from %2$s to %3$s compared to %4$s. Based on this an evolution of each row of %5$s%% is expected.",
-        "TitleRowChangeDetails": "'%1$s' changed from %2$s (%3$s) to %4$s (%5$s) %6$s.",
-        "TitleRowDisappearedDetails": "'%1$s' decreased by %2$s and disappeared in %3$s compared to %4$s.",
-        "TitleRowMoverAndShaker": "This row had a higher impact than the average.",
+        "TitleConsideredInsightsChanges": "These rows increased or decreased by at least %1$s visits (%2$s%% of %3$s total visits).",
+        "TitleConsideredInsightsGrowth": "These rows grew at least %1$s%% compared to %2$s.",
+        "TitleConsideredMoversAndShakersChanges": "Considered movers only with growth of more than %1$s%% visits or falling by more than %2$s%%. As new entries only if increasing by more than %3$s%% (%4$s), and disappeared rows if shrinking by fewer than %5$s%% (%6$s).",
```

Comment:
```suggestion
        "TitleConsideredMoversAndShakersChanges": "Movers are those growing more than the %1$s% percentile of visits, or falling by more than the bottom %2$s% percentile.\nEverything below 5% of the total amount of visits (%5$s%% individually, which is %4$s visits within the period) disappears, and appears as new entries again if making it above this threshold (by growing more than %3$s%% individually, which is %4$s visits within the period).",
```

Reply:
I'd suggest to use entries (or rows), as it might be any dimension (like pages, browsers, operating systems, countries, ...)

Reply:
I'm still struggling with this one. I think it might mean what it should, but I'm not fully sure.
@justinvelluppillai would you understand what this should mean:

> Movers are those entries growing more than the 84.1% percentile, or falling by more than the bottom -84.1% percentile when ordered by individual growth.
> Everything below 5% of the total amount of visits (5% individually, which is 117 visits within the period) disappears, and appears as new entries only and every time they make it above this threshold (by growing more than 7% individually, which is 164 visits within the period).

Reply:
@sgiehl that is not easily understood. I think the existing version is slightly more understandable here.